tests/csm/rest/test_query_deployment.py

```diff
@@ -80,6 +92,8 @@ def setup_method(self):
         resp = self.ha_obj.poll_cluster_status(self.csm_obj.master)
         assert resp[0], resp[1]
         self.log.info("Cleanup: Cluster status checked successfully")
+        deploy_end_time = time.time()
```

Comment:
Is this required?

Reply:
it is required to Verify deployment time between start and end deployment time in some tests hence added.

Reply:
this should be self.deploy_end_time instead of deploy_end_time

Reply:
ok noted

Reply:
done